src/components/ForecastDetails.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/ForecastDetails.css";
+
+function ForecastDetails({ forecast }) {
+  const { date, humidity, temperature, wind } = forecast;
```

Comment:
Good use of destructuring here. You can also do further destructuring in the brackets if you want. Like this:
`function ForecastDetails({ forecast: { date, humidity, temperature, wind } }) {`
but I don't think it's very readable in my opinion. I think the way you have done it here is betetr